src/program/lwaftr/tests/data/icmp_endaddr.conf

```diff
@@ -42,27 +42,37 @@ softwire-config {
     error-rate-limiting {
       packets 600000;
     }
-    ip 10.10.10.10;
-    mac 12:12:12:12:12:12;
     mtu 2000;
-    next-hop {
-      mac 68:68:68:68:68:68;
-    }
     reassembly {
       max-fragments-per-packet 40;
     }
   }
+  instance {
+    device test;
```

Comment:
In a followup I think we should change this.  I think I pushed you into making what in hindsight looks like a wrong decision, to make the PCI device be the "key" to identify an instance and then have queues inside that.  Having to specify a PCI address when there is simply no sensible value as in here is a bit wonky.  Additionally code that does e.g.

```lua
+   local inst_configs = lwutil.produce_instance_configs(conf)
+   local device, lwaftr_config = next(inst_configs)
+   local queue = lwaftr_config.softwire_config.instance[device].queue.values[1]
```

I find this really weird to use the noun "queue" to reflect what is really a lwAFTR instance.  I know how it comes about but I think that reflects that we're organizing things wrong; perhaps we should have instead

```yang
list instance {
   key name;
   leaf name { type string; }
   leaf device { type string; }
   leaf queue { type uint8; default 0; }
   container external-interface { ... }
   ...
}
```

This allows us to refer to instances simply by name.  If we need a PCI address or queue number, that's up to `setup.lua` to assert.  Otherwise it's possible to run without these, as in `bench` etc...  Dunno.  Definitely followup material though.  Already having done what you have to this point is good, if we decide to migrate again (wdyt about that?) we can do that in a followup.